pkg/internal/cluster/create/actions/config/config.go

```diff
@@ -119,11 +119,42 @@ func (a *Action) Execute(ctx *actions.ActionContext) error {
 		}
 	}
 
-	// Create the config in all nodes concurrently
+	// Create the kubeadm config in all nodes concurrently
 	if err := errors.UntilErrorConcurrent(fns); err != nil {
 		return err
 	}
 
+	// if we have containerd config, patch all the nodes concurrently
+	if len(ctx.Config.ContainerdConfigPatches) > 0 {
```

Comment:
also check for ContainerdConfigPatchesJSON6902 ?

Reply:
done, thanks :-)